app/js/components/identifiers/identifiers.jsx

```diff
@@ -316,135 +322,135 @@ export default class Identifiers extends React.Component {
-                                  </option>
-                                ))
-                              }
-                            </select>
-                          </div>
-                          <div className="arrange-horizontally">
-                            <h6><b>Created By </b></h6>
-                            <h5 name="creator">
-                              {id.auditInfo.creator.display} on {'  '}
-                              {new Date(id.auditInfo.dateCreated).toString()}
-                            </h5>
                           </div>
-                          {!id.preferred && !id.voided &&
-                            <div className="arrange-horizontally">
-                              <h6><b>Preferred  </b></h6> <t />
+                          <div className="card-body">
+                            <div
+                              className={
+                                this.state.editErrors.uuid === id.uuid ?
+                                  editErrorClass : ''
+                              }>
+                              <h6><b>Identifier</b></h6>
                               <input
-                                className="form-check-input"
-                                type="Checkbox"
-                                name="preferred"
-                                defaultChecked={id.preferred}
+                                className="form-control col-sm-8"
+                                type="text"
+                                name="identifier"
+                                defaultValue={
+                                  this.state.activeCard !== index
+                                    ? id.identifier
+                                    : editIdentifiers.identifier ||
+                                    id.identifier
+                                }
+                                onChange={e => this.handleChange(e, index)}
                                 disabled={id.voided}
-                                onChange={() => this.handlePreferred(id.preferred)}
                               />
+                              {this.state.editErrors.uuid === id.uuid &&
+                                <div className="input">{editError}</div>}
+                            </div>
+                            <div>
+                              <div>
+                                <h6><b>Identifier Type</b></h6>
+                                <select
+                                  className="form-control"
+                                  name="identifierType"
+                                  value={editIdentifiers.identifierType ||
+                                    id.identifierType.display}
+                                  disabled={id.voided}
+                                  onChange={e => this.handleChange(e, index)}
+                                >
+                                  {
```

Comment:
For this and probably some others, lets use event instead of e so that it is more readable and removes needs for any comments. Check the guidelines in this article https://improvingsoftware.com/2011/06/27/5-best-practices-for-commenting-your-code/

Reply:
That is a very good one @jacksono 👍

Reply:
👍

Reply:
I'll change that, It wasn't part of my contribution though.